api/data_explorer/dataset_faceted_search.py

```diff
@@ -50,26 +50,33 @@ def get_dataset_name():
         return dataset['name']
 
 
+def get_table_names():
+    """Gets a list of table names from facet_fields.csv.
+
+    current_app.config['DATASET_CONFIG_DIR'] must be set before this is called.
```

Comment:
Add somewhere:

Table names are fully qualified: `<project id>:<dataset id>:<table name>`

Returned list is sorted alphabetically

Reply:
Done